scripts/build.sh

```diff
@@ -282,5 +282,3 @@ if [ ! -f "${SCRIPT_DIR}/../openssl-${OPENSSL_VERSION}.tar.gz" ]; then
 fi
 
 build_ios
-build_macos
-build_catalyst
```

Comment:
In my testing, dropping macOS and Catalyst builds cut the build time in half, from about 1 hour to about 30 minutes 😅